### ISSUE TITLE
the preprocessing process of skeleton data

### ISSUE THREAD

**Issue by Leoric1** [hulianyuyy/DSTA-SLR#3]
Could you share the preprocessing process of skeleton data?

**Comment by hulianyuyy** [comment]
Generally, we follow [this guide](https://github.com/jackyjsy/SAM-SLR-v2/tree/main/SL-GCN) to process our skeleton data for fair comparison.

**Comment by snalyami** [comment]
Thank you for sharing you code. I'm currently extracting the keypoints using demo.py in the guide you mentioned, but i noticed this issue [here](https://github.com/jackyjsy/data-prepare/issues/4). I'm worried that the code has a typo as indicated by the user. Did you use this code? or should it be fixed to 

`def merge_hm(hms_list):
    assert isinstance(hms_list, list)
    for hms in hms_list:
        hms[1,:,:,:] = torch.flip(hms[1,index_mirror,:,:], [2])
    
    hm = torch.cat(hms_list, dim=0)
   # previously hm = torch.mean(**hms**, dim=0)
    hm = torch.mean(**hm**, dim=0)
    print(hm)
    return hm`

Many thanks for your help!

**Comment by hulianyuyy** [comment]
Yes, it can be fixed as this.
